app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue

```diff
@@ -348,6 +349,7 @@ export default {
       v-if="showContextMenu"
       :x="contextMenu.x"
       :y="contextMenu.y"
+      :scroll-lock-element="scrollLockElement"
```

Comment:
Can this not use it's parent or something?

Reply:
@scmmishra I initially tried using `conversationListRef` since it's the parent element, but it didn’t work because of the `DynamicScroller`. So, I tried this way, as the scrolling works correctly within the `DynamicScroller`.